workload/aspnetapp.yaml

```diff
@@ -125,7 +125,10 @@ spec:
     http:
       paths:
       - path: /
+        pathType: Prefix
         backend:
-          serviceName: aspnetapp-service
-          servicePort: http
+          service:
+            name: aspnetapp-service
+            port:
+              number: 80
```

Comment:
is now the preference using the port number instead of `servicePort: http`?

Reply:
That's correct, that's part of the new `v1` and `v1beta1` is deprecated in 1.19

Reply:
Probably just to be more precise/intentional